pytorch_lightning/callbacks/progress.py

```diff
@@ -291,10 +291,10 @@ def init_validation_tqdm(self) -> tqdm:
         )
         return bar
 
-    def init_test_tqdm(self) -> tqdm:
+    def init_test_tqdm(self, trainer=None) -> tqdm:
         """ Override this to customize the tqdm bar for testing. """
         bar = tqdm(
-            desc='Testing',
+            desc="Predicting" if trainer.is_predicting else "Testing",
```

Comment:
Might be nitpicking, but do we need the default value `trainer=None` here? because when `trainer is None`, it will raise `AttributeError`.
```
AttributeError: 'NoneType' object has no attribute 'is_predicting'
```

Reply:
So, I'm thinking we should remove the default value or check if `trainer` has `is_predicting` before using `trainer.is_predicting`.